### PR TITLE
#10881: update golden function

### PR DESCRIPTION
**Pull request by bharane-ab**
### Ticket
Link to Github Issue #10881 

### Problem description
Update golden function for test_unary.py and test_activation.py

### Checklist
- [x] Post commit CI passes
- [x] Model regression CI testing passes (if applicable)
- [ ] New/Existing tests provide coverage for changes

### CODE REVIEW

**Review by eyonland — CHANGES_REQUESTED** [review]
Please have this reviewed by an MCW colleague prior to posting the PR.

File: tests/ttnn/unit_tests/operations/test_unary.py [hunk 1]
```diff
@@ -320,34 +325,32 @@ def test_logit(device, h, w, scalar):
...
 def test_pow(device, h, w, scalar):
-    run_unary_test_with_float(device, h, w, scalar, ttnn.pow, torch.pow, pcc=0.9)
+    run_unary_test_with_float(device, h, w, scalar, ttnn.pow, pcc=0.9)
 
 
-@pytest.mark.parametrize("scalar", [-0.5, -2.0])
 @pytest.mark.parametrize("h", [64])
 @pytest.mark.parametrize("w", [128])
 def test_relu_min(device, h, w, scalar):
-    run_unary_test_with_float(device, h, w, scalar, ttnn.relu_min, torch_relu_min)
```
Comment (KalaivaniMCW) [inline tests/ttnn/unit_tests/operations/test_unary.py:330]: any reason why we are removing the scalar test param?

Comment (KalaivaniMCW) [reply tests/ttnn/unit_tests/operations/test_unary.py:330]: need to pass upper_limit and lower_limit as kwargs to the golden function. so torch_relu_min and torch_relu_max can have their own test and separate golden function.

**Review by ayerofieiev-tt — APPROVED** [review]

File: ttnn/ttnn/operations/unary.py [hunk 2]
```diff
@@ -26,17 +21,6 @@ def torch_multigammaln(x, *args, **kwargs):
         result += 3.434189657547
         return result
 
-    def torch_prelu(x, *args, **kwargs):
```
Comment (KalaivaniMCW) [inline ttnn/ttnn/operations/unary.py:29]: add corresponding attach_golden_function for this

File: tests/ttnn/unit_tests/operations/test_unary.py [hunk 3]
```diff
@@ -29,15 +31,16 @@ def run_unary_test(device, h, w, ttnn_function, torch_function, pcc=0.9999):
 
 def run_identity_test(device, h, w, data_type, pcc=0.9999):
     torch.manual_seed(0)
-
+    ttnn_function = ttnn.identity
```
Comment (eyonland) [inline tests/ttnn/unit_tests/operations/test_unary.py:34]: What is the purpose of this variable?

Comment (bharane-ab) [reply tests/ttnn/unit_tests/operations/test_unary.py:34]: It is used to get the golden function for identity op.

**Review by eyonland — APPROVED** [review]

**Pull request merged**